pages/Profile/index.tsx

```diff
@@ -1,13 +1,49 @@
-import React from "react";
+import { style } from '@mui/system';
+import React from 'react';
 import Layout from "../../components/Layout/Layout";
 import styles from "./Profile.module.css";
 
 const ProfileSettingsPage: React.FunctionComponent = () => {
-  return (
-    <Layout title="Profile">
-      <main className={styles.main}>Profile Settings</main>
-    </Layout>
-  );
-};
+    
+    return (
+        <body className = {styles.body}>
```

Comment:
Wrapping the Layout component within the `body` tag shouldn't be necessary. We will work on styling this Layout component more!